server/util/validation_decorators/validate_image_files.py

```diff
@@ -0,0 +1,38 @@
+from flask import jsonify, request
+from functools import wraps
+
+
+def validate_files(f):
+    @wraps(f)
+    def wrapper(*args, **kwargs):
+        # https://flask.palletsprojects.com/en/1.1.x/patterns/fileuploads/
+        # request.files is ImmutableMultiDict([('image', <FileStorage: 'example2.png' ('image/png')>)])
+
+        # File of key image has to exist
+        if 'image' not in request.files:
+            return jsonify({"error": "file part not present"}), 400
+
+        # If filename doesn't exist
+        files = request.files.getlist("image")
+        for file in files:
+            if file.filename == '':
+                return jsonify({"error": "No selected file"}), 400
+
+        if request.form["project_id"] != "null":
```

Comment:
```suggestion
        if request.form["folder"] == "project":
```
We should check that the client provided a project id if they are saving the file to a project.